st2common/st2common/models/api/action.py

```diff
@@ -497,6 +497,27 @@ class ActionAliasAPI(BaseAPI, APIUIDMixin):
                 "type": "array",
                 "items": {"type": "string"},
                 "description": "Possible parameter format."
+            },
+            "aliases": {
```

Comment:
The one example I have (not very characteristic, but still) is a command getting a nearest event from the calendar: "what's next?"

I've seen people writing it as "what is next?" and "what's next" very often, which is one concern, and the other is the single quote in "what's", which Slack changes to proper apostrophe, while IRC, for example, will not, so I need to match both.

At the same time, I want `!help` to provide a clear and meaningful list, not a list of the same command with typos and variations. So that's what I do:
```
formats:
    - "what's next?"
    - "when is {{ query }}?"
aliases:
    - "what(.s| is) next\\??"
    - "when is {{ query }}\\??"
```
In `formats` I have commands that are also help entries, in `aliases` I do a wider match with a regex, covering all possible variations—and putting it in `!help` will not be user-friendly in the slightest. So, I really can't think of a name other than `aliases` because that's exactly what it is. Still, aliases for aliases is kind of weird, and I'm open to suggestions on this one.